k8s/templates/business-create-ui.dc-srvc-rt-cmap-kcmap.json

```diff
@@ -310,7 +329,7 @@
             "value": ""
         },
         {
-            "name": "KEYCLOAK_CONFIG_PATH",
+            "name": "KEYCLOAK_CONFIG_URL",
```

Comment:
"KEYCLOAK_CONFIG_PATH" is the current name of this key.

Reply:
Oops! I'll update it